### DISCUSSION TITLE
Rename this integration?

### DISCUSSION THREAD

**Discussion by WillCodeForCats** [WillCodeForCats/solaredge-modbus-multi#27] [General]
Consider renaming this integration to something less SolarEdge specific if it's going to support more than just SolarEdge inverters.

I don't really like the current name anyway but couldn't think of anything better.

**Comment by dhover** [comment]
A suggestion: inverter-modbus-multi

**Comment by WillCodeForCats** [comment]
I'd thought I was going to expand and go more Sunspec generic, but since there's another HACS integration focusing on that I'm not going to duplicate that work. SolarEdge what I have at home, so I will personally continue to maintain this integration for my Home Assistant.
